### PR TITLE
Npm build issues fix

### PR DESCRIPTION
**Pull request by kingman**
Hi, @ptone 
by making the two changes,
I was able to make the 'npm install' working again.

I'll refactor more later on by using the: @google-cloud/iot library

### CODE REVIEW

**Comment by ToddKopriva** [comment]
@ptone , could you take a look at these proposed changes?

File: tutorials/cloud-iot-firestore-config/functions/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   "devDependencies": {
     "gts": "^0.5.4",
     "tslint": "^5.8.0",
-    "typescript": "~2.6.1"
+    "typescript": "~2.8.3"
```
Comment (ptone) [inline tutorials/cloud-iot-firestore-config/functions/package.json]: we might as well upgrade to the 3x latest branch and test - it should all still work

File: tutorials/cloud-iot-firestore-config/functions/src/index.ts [hunk 2]
```diff
@@ -21,7 +21,6 @@ import cbor = require('cbor');
 
 import * as admin from "firebase-admin";
 import * as functions from 'firebase-functions';
-import { runInDebugContext } from 'vm';
```
Comment (ptone) [inline tutorials/cloud-iot-firestore-config/functions/src/index.ts]: +1  - this is certainly a legacy construct at this point

File: tutorials/cloud-iot-firestore-config/functions/package.json (hunk 1 above)
Comment (kingman) [reply tutorials/cloud-iot-firestore-config/functions/package.json]: I tried with 3x version but it had some other issues.

**Comment by kingman** [comment]
@ptone 
Lets merge this asap for now, I shared this with some CE:s. and they gonna ref to this for a coming take5 episode. Really need the setup working. I can fix the other things later on.

**Comment by ToddKopriva** [comment]
I'll merge this and publish as soon as I see the review approval from @ptone.

**Comment by ptone** [comment]
LGTM
I was able to `firebase deploy` this branch with no errors.

FYI - @kingman in the future please submit PRs from a branch on your fork instead of a branch on this repo.

**Comment by ToddKopriva** [comment]
I'll publish these changes soon. Publishing lags a little bit, since our build process is not instantaneous.

**Pull request merged**